package.json

```diff
@@ -85,6 +88,7 @@
     "jshint": "2.8.0",
     "lab": "6.0.0",
     "seneca-echo": "0.2.0",
-    "seneca-error-test": "0.2.2"
+    "seneca-error-test": "0.2.2",
+    "serve": "^1.4.0"
```

Comment:
This is fine for now... will merge.  But we should pin to a specific version like the other dependencies.